<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns='http://www.w3.org/1999/xhtml' xml:lang='en' lang='en'>
<head>
<title>meta-cgl - Enable Carrier Grade Linux compliance through a reference Linux distribution
</title>
<meta name='generator' content='cgit v0.10.2'/>
<meta name='robots' content='index, nofollow'/>
<link rel='stylesheet' type='text/css' href='/cgit/cgit.css'/>
<link rel='shortcut icon' href='/favicon.ico'/>
<link rel='alternate' title='Atom feed' href='http://git.yoctoproject.org/cgit/cgit.cgi/meta-cgl/atom/meta-cgl-common/recipes-cgl/makedumpfile/files/alias-powerpc-powerpc32.patch?h=master' type='application/atom+xml'/>
<script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1.3/jquery.min.js"></script>
<script type="text/javascript">
$(document).ready(function() {
$('.logo').append('<a href="http://git.yoctoproject.org/clean/cgit.cgi/' + window.location.pathname.substring(window.location.pathname.lastIndexOf("cgit.cgi")+9) + '">light theme</a>');
});
</script>
</head>
<body>
<div id='cgit'><table id='header'>
<tr>
<td class='logo' rowspan='2'><a href='/cgit/cgit.cgi/'><img src='/cgit/yocto-project-transp.png' alt='cgit logo'/></a></td>
<td class='main'><a href='/cgit/cgit.cgi/'>index</a> : <a title='meta-cgl' href='/cgit/cgit.cgi/meta-cgl/'>meta-cgl</a></td><td class='form'><form method='get' action=''>
<select name='h' onchange='this.form.submit();'>
<option value='master' selected='selected'>master</option>
</select> <input type='submit' name='' value='switch'/></form></td></tr>
<tr><td class='sub'>Enable Carrier Grade Linux compliance through a reference Linux distribution
</td><td class='sub right'>git repository hosting</td></tr></table>
<table class='tabs'><tr><td>
<a href='/cgit/cgit.cgi/meta-cgl/about/'>about</a><a href='/cgit/cgit.cgi/meta-cgl/'>summary</a><a href='/cgit/cgit.cgi/meta-cgl/refs/'>refs</a><a href='/cgit/cgit.cgi/meta-cgl/log/meta-cgl-common/recipes-cgl/makedumpfile/files/alias-powerpc-powerpc32.patch'>log</a><a class='active' href='/cgit/cgit.cgi/meta-cgl/tree/meta-cgl-common/recipes-cgl/makedumpfile/files/alias-powerpc-powerpc32.patch'>tree</a><a href='/cgit/cgit.cgi/meta-cgl/commit/meta-cgl-common/recipes-cgl/makedumpfile/files/alias-powerpc-powerpc32.patch'>commit</a><a href='/cgit/cgit.cgi/meta-cgl/diff/meta-cgl-common/recipes-cgl/makedumpfile/files/alias-powerpc-powerpc32.patch'>diff</a><a href='/cgit/cgit.cgi/meta-cgl/stats/meta-cgl-common/recipes-cgl/makedumpfile/files/alias-powerpc-powerpc32.patch'>stats</a></td><td class='form'><form class='right' method='get' action='/cgit/cgit.cgi/meta-cgl/log/meta-cgl-common/recipes-cgl/makedumpfile/files/alias-powerpc-powerpc32.patch'>
<select name='qt'>
<option value='grep'>log msg</option>
<option value='author'>author</option>
<option value='committer'>committer</option>
<option value='range'>range</option>
</select>
<input class='txt' type='text' size='10' name='q' value=''/>
<input type='submit' value='search'/>
</form>
</td></tr></table>
<div class='path'>path: <a href='/cgit/cgit.cgi/meta-cgl/tree/'>root</a>/<a href='/cgit/cgit.cgi/meta-cgl/tree/meta-cgl-common'>meta-cgl-common</a>/<a href='/cgit/cgit.cgi/meta-cgl/tree/meta-cgl-common/recipes-cgl'>recipes-cgl</a>/<a href='/cgit/cgit.cgi/meta-cgl/tree/meta-cgl-common/recipes-cgl/makedumpfile'>makedumpfile</a>/<a href='/cgit/cgit.cgi/meta-cgl/tree/meta-cgl-common/recipes-cgl/makedumpfile/files'>files</a>/<a href='/cgit/cgit.cgi/meta-cgl/tree/meta-cgl-common/recipes-cgl/makedumpfile/files/alias-powerpc-powerpc32.patch'>alias-powerpc-powerpc32.patch</a></div><div class='content'>blob: e918ce87747e032e9e4f43d3c6c5b41cbe7b9140 (<a href='/cgit/cgit.cgi/meta-cgl/plain/meta-cgl-common/recipes-cgl/makedumpfile/files/alias-powerpc-powerpc32.patch'>plain</a>)
<table summary='blob content' class='blob'>
<tr><td class='linenumbers'><pre><a id='n1' href='#n1'>1</a>
<a id='n2' href='#n2'>2</a>
<a id='n3' href='#n3'>3</a>
<a id='n4' href='#n4'>4</a>
<a id='n5' href='#n5'>5</a>
<a id='n6' href='#n6'>6</a>
<a id='n7' href='#n7'>7</a>
<a id='n8' href='#n8'>8</a>
<a id='n9' href='#n9'>9</a>
<a id='n10' href='#n10'>10</a>
<a id='n11' href='#n11'>11</a>
<a id='n12' href='#n12'>12</a>
<a id='n13' href='#n13'>13</a>
<a id='n14' href='#n14'>14</a>
<a id='n15' href='#n15'>15</a>
<a id='n16' href='#n16'>16</a>
<a id='n17' href='#n17'>17</a>
<a id='n18' href='#n18'>18</a>
</pre></td>
<td class='lines'><pre><code>Create Alias for target for powerpc as powerpc32

Signed-off-by: Siva Borra &lt;siva.borra@enea.com&gt;
Upstream-status: Pending

diff -rupN makedumpfile-1.5.8/Makefile makedumpfile-1.5.8/Makefile
--- makedumpfile-1.5.8/Makefile	2015-03-24 02:58:33.000000000 +0100
+++ makedumpfile-1.5.8/Makefile	2015-06-23 11:08:30.595655336 +0200
@@ -25,7 +25,8 @@ endif
 ARCH := $(shell echo ${TARGET}  | sed -e s/i.86/x86/ -e s/sun4u/sparc64/ \
 			       -e s/arm.*/arm/ -e s/sa110/arm/ \
 			       -e s/s390x/s390/ -e s/parisc64/parisc/ \
-			       -e s/ppc64/powerpc64/ -e s/ppc/powerpc32/)
+			       -e s/ppc64/powerpc64/ -e s/ppc/powerpc32/ \
+			       -e s/powerpc/powerpc32/)
 
 CROSS :=
 ifneq ($(TARGET), $(HOST_ARCH))
</code></pre></td></tr></table>
</div> <!-- class=content -->
<div class='footer'>generated  by cgit v0.10.2 at 2015-10-19 14:35:52 (GMT)</div>
</div> <!-- id=cgit -->
</body>
</html>
